doc-surrealdb_versioned_docs/version-1.x/surrealql/statements/index.mdx

```diff
@@ -0,0 +1,13 @@
+# Statements
+
+SurrealDB has a variety of statements that let you configure and query a database: `CREATE`, `INSERT`, `UPDATE`, and so on.
+
+## Statement parameters
+
+A number of parameters prefixed with `$` are automatically available within a statement that provide access to relevant context inside the statement. These are known as reserved variable names. For example:
+
+* `$before` and `$after` can be accessed in statements that mutate values to see the values before and after an update,
```

Comment:
suggestion: add links to each parameter via anchor

Reply:
Good idea! Will do that.